packages/core/src/util/env-var.ts

```diff
@@ -1,14 +1,23 @@
 import { MetriportError } from "./error/metriport-error";
```

Comment:
Dont we need this file in core because the lambdas? Or the lambdas can access shared?

Reply:
Yeah, lambdas have access to `shared`. Will add an item to the release plan to test that too.

Reply:
There was no file being moved out of core and no updates to lambda, though.